components/bluetooth/lib.rs

```diff
@@ -953,10 +953,13 @@ impl BluetoothManager {
                     false => c.stop_notify(),
                 };
                 match result {
+                    // Step 11.
```

Comment:
We'll need a comment containing a link to the appropriate specification, otherwise these steps are mysterious :)